python-tic-tac-toe-game-tkinter/source_code_step_1/tic_tac_toe.py

```diff
@@ -0,0 +1,52 @@
+"""A Tic Tac Toe game built with Python and Tkinter."""
```

Comment:
:writing_hand: **Writing Tip**

Your individual steps have a docstring that refers to tic-tac-toe without hyphens.